h2o-r/h2o-package/R/models.R

```diff
@@ -4712,7 +4712,7 @@ h2o.genericModel <- function(mojo_file_path){
 #' original_model <- h2o.gbm(x=features, y = "Species", training_frame = data)
 #'
 #' # Download the trained GBM model as MOJO (temporary directory used in this example)
-#' mojo_original_name <- h2o.download_mojo(model = original_model, path = tempdir())
+#' mojo_original_name <- h2o.saveMojo(original_model, path = tempdir())
```

Comment:
should be save_mojo

Reply:
This is R. R has saveMojo and download_mojo methods.

Don't ask me why, the API is simply done this way. Probably because the API went from cameCase to Tidyverse's snake_case at one point in time and various people started enforcing snake_case.

Reply:
That's also why this is different in documentation for Python and R. I got confused too. What we should do is to unite this behavior and simply delegate the method calls. WDYT cc @michalkurka ?

Reply:
yeah, maybe we should have both - can we discuss it on #h2o-3-api-updates?

Reply:
Will do.

Reply:
So, after quick discussion in `h2o-api-changes`, I introduced `h2o.save_mojo` in `export.R`. Calls from `h2o.saveMojo` are now delegated to `h2o.save_mojo`. Any call to `h2o.saveMojo` will pass but will throw a deprecation warning, as `.Deprecated("h2o.save_mojo")` is called before the call is delegated.

- Changed in documentation in `save-and-load-model.rst` @angela0xdata please check.
- Introduced a new quick test to make sure the method `h2o.saveModel` works.
- Testing `h2o.save_model` by using it in the generic mojo conveniencem methods test.